server/session/session.go

```diff
@@ -104,6 +105,12 @@ type Conn interface {
 	StartGame(data minecraft.GameData) error
 }
 
+// actorIdentifier represents the structure of an actor identifier sent over the network.
+type actorIdentifier struct {
```

Comment:
I'd also move this down to where it's used. You could even have it inside of the sendAvailableEntities method.